authentication/utils.py

```diff
@@ -67,17 +67,22 @@ def load_drf_strategy(request=None):
     )
 
 
+def get_hash_object(value):
```

Comment:
why are we returning hash_object when we can simply return the hash (hash_object.hexdigest())? and the name of function can be changed a bit to get more specific:
```suggestion
def get_md5_hash(value):
```